src/styles/optionAlternateTextPlatformStyles/types.ts

```diff
@@ -0,0 +1,5 @@
+type OptionAlternateTextPlatformStyles = {
+    paddingTop?: number;
+};
```

Comment:
```suggestion
type OptionAlternateTextPlatformStyles = Partial<TextStyle>;
```

This style is related to text styles, we can type in this way.

Reply:
I was following documentation requirements 
> Strive to type as strictly as possible.

But I think that styles aren't that crucial in terms of types, so we may just as well use TextStyle :)

Reply:
Yes I understand 😄 We can restrict a little bit with this:

```ts
type OptionAlternateTextPlatformStyles = Partial<Pick<TextStyle, 'paddingTop'>>;
```

@BartoszGrajdek WDYT?

Reply:
@fabioh8010 this looks better, let's go with it 😄